search/lib/convert/granules.js

```diff
@@ -180,7 +180,7 @@ async function cmrGranuleToStac (event, granule) {
       granule.links.filter(l => l.rel === BROWSE_REL)
     );
     opendapLink = _.first(
-      granule.links.filter(l => l.rel === DOC_REL && !l.inherited && l.href.includes('opendap'))
+      granule.links.filter(l => l.rel === SERVICE_REL && !l.inherited && l.href.includes('opendap'))
```

Comment:
According to CMR-7795, the opendap link is defined as:
```
4) Only the first opendap link is included in stac assets
opendap links are identified by atom links with type "service".
```
Note, there is no requirement that the URL must have `opendap` in it.